### PR TITLE
[Merged by Bors] - feat: add examples to docstrings

### PR DESCRIPTION
**Pull request by kbuzzard**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

The "natural transformation to the constant functor" thing always throws me; I've added the simple example which I am often imagining when I'm using this stuff.

### CODE REVIEW

File: Mathlib/CategoryTheory/Limits/Cones.lean [hunk 1]
```diff
@@ -152,6 +156,10 @@ theorem Cone.w {F : J ⥤ C} (c : Cone F) {j j' : J} (f : j ⟶ j') :
 * an object `c.pt` and
 * a natural transformation `c.ι : F ⟶ c.pt` from `F` to the constant `c.pt` functor.
 
+For example, if the source `J` of `F` is a partially ordered set, then to give
+`c : Cocone F` is to give a collection of morphisms `ιⱼ : F j ⟶ c.pt` and, for
+all `j ≤ k` in `J`, morphisms `ιⱼₖ : F j ⟶ F k` such that `Fⱼₖ ≫ Fₖ = Fⱼ` for all `j ≤ k`.
```
Comment (jcommelin) [inline Mathlib/CategoryTheory/Limits/Cones.lean:161]: Out of curiosity, why do you use `i \le j` in the example above, and `j \le k` here?

Comment (semorrison) [reply Mathlib/CategoryTheory/Limits/Cones.lean:161]: Maybe it is to avoid visual confusion with iota?

**Comment by semorrison** [comment]
bors d+

File: Mathlib/CategoryTheory/Limits/Cones.lean (hunk 1 above)
Comment (kbuzzard) [reply Mathlib/CategoryTheory/Limits/Cones.lean:161]: Yes it was simply that

**Pull request closed without merging**